pontoon/teams/static/css/team.css

```diff
@@ -123,6 +123,27 @@
   color: #7BC876;
 }
 
+.info {
+  text-align: center;
```

Comment:
We should keep the default alignment (left). That's what users expect by default. And that's how we currently align team and project information blocks.